l10n_br_cnpj_search/models/receitaws_webservice.py

```diff
@@ -61,7 +61,7 @@ def _receitaws_get_phones(self, data):
         If there is more than one phone, the second is assigned to mobile."""
         phone = False
         mobile = False
-        if data.get("telefone") != "":
+        if data.get("telefone", "") != "":
```

Comment:
Dessa forma não daria certo?
```suggestion
        if data.get("telefone"):
```

Reply:
puts pior que eu eu vi que eu tinha testado mesmo num terminal e interpretei de forma errada o teste, e achei estranho mesmo... Preciso de sono. Ja arrumo.

Reply:
arrumei